gateway/migrations/0001_custom_domains.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS custom_domains (
+  fqdn TEXT PRIMARY KEY,
+  project_name TEXT NOT NULL,
```

Comment:
It might be worth adding a FK reference to this column so that we don't end up with dangling entries.